eng/Testing.targets

```
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup Condition=" '$(SkipTests)' == '' and '$(IsTestProject)' == 'true' ">
```

```suggestion
  <PropertyGroup Condition=" '$(SkipTests)' != 'true' and '$(IsTestProject)' == 'true' ">
```
Super minor NIT but this way it will still work even if someone happens to do `/p:SkipTests=false`

---

Oh nvm, I guess I was misunderstanding this.